### ISSUE TITLE
mods not loading

### ISSUE THREAD

**Issue by Krogine** [soulsmods/ModEngine2#40]
i put the mods in the mod folder and hit the .bat file and even though the game starts up none of the mods are present at all

**Comment by garyttierney** [comment]
Do you have a modengine log file in the folder containing your modengine config file?

**Comment by PaulsgamingLive** [comment]
2022-04-09 21:19:57.796] [modengine] [info] Enabling extension base
[2022-04-09 21:19:57.796] [modengine] [info] Enabled extension base
[2022-04-09 21:19:57.796] [modengine] [info] Enabling extension mod_loader
[2022-04-09 21:19:57.796] [modengine] [info] Installing mod location mod
[2022-04-09 21:19:57.796] [modengine] [info] Resolved mod path to E:\ModEngine-2.0.0-preview3-win64 (2)\ModEngine-2.0.0-preview3-win64\mod
[2022-04-09 21:19:57.796] [modengine] [info] Enabled extension mod_loader
[2022-04-09 21:19:57.841] [modengine] [info] Applied 3 hooks
[2022-04-09 21:19:57.842] [modengine] [info] Starting worker thread
[2022-04-09 21:28:51.942] [modengine] [info] Destroying
[2022-04-09 21:28:51.942] [modengine] [info] Destroying
[2022-04-09 21:28:51.942] [modengine] [info] Destroying
[2022-04-09 21:28:51.942] [modengine] [info] Destroying
[2022-04-09 21:28:51.942] [modengine] [info] Destroying
[2022-04-09 21:28:51.942] [modengine] [info] Destroying
[2022-04-09 21:28:51.942] [modengine] [info] Destroying
[2022-04-09 21:28:51.950] [modengine] [info] Destroying
[2022-04-09 21:28:51.950] [modengine] [info] Destroying
[2022-04-09 21:28:51.950] [modengine] [info] Destroying


When I load it I get this, Is it because I was using UXM?

**Comment by garyttierney** [comment]
> When I load it I get this, Is it because I was using UXM?

I don't think that should cause any problems, but would you mind trying with a non-UXM executable to see if it gets you any further?
